contracts/upgradeable_contracts/BaseFeeManager.sol

```diff
@@ -12,16 +12,22 @@ contract BaseFeeManager is EternalStorage, FeeTypes {
     event HomeFeeUpdated(uint256 fee);
     event ForeignFeeUpdated(uint256 fee);
 
+    uint256 internal constant MAX_FEE = 1 ether;
```

Comment:
consider to add a description explaining that `1 ether` is not a fee the bridge can gather as a maximum but some relative value used to calculate the fee percentage.

Reply:
Thanks for suggestion, added a comment https://github.com/poanetwork/poa-bridge-contracts/pull/209/commits/53f2a1ceaaf2316573af12244c24e428d6d843d7